src/app/components/user/cognito-user-invite.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  makeStyles,
+} from '@material-ui/core';
+import PersonAddIcon from '@material-ui/icons/PersonAdd';
+import EmailIcon from '@material-ui/icons/Email';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+import theme from '../../themes/theme';
+import { ChplDialogTitle, ChplTooltip, ChplTextField } from '../util';
+
+const useStyles = makeStyles({
+  content: {
+    display: 'grid',
+    gap: '8px',
+  },
+  iconSpacing: {
+    marginLeft: '4px',
+  },
+});
+
+const validationSchema = yup.object({
+  email: yup.string()
+    .required('Email is required')
+    .email('Enter a valid Email'),
+});
+
+function ChplCognitoUserInvite(props) {
+  const [open, setOpen] = React.useState(false);
```

Comment:
```suggestion
  const [open, setOpen] = useState(false);
```

You're already importing the `useState`. No need to prefix it with `React`, and we don't do that anywhere else